app/controllers/api/series_images_controller.rb

```diff
@@ -6,6 +6,11 @@ class Api::SeriesImagesController < Api::BaseController
   filter_resources_by :series_id
 
   announce_actions decorator: Api::Msg::ImageRepresenter, subject: :image
+  announce_actions :update, resource: :series_resource
```

Comment:
currently both the Image Callback Worker and the Audio Callback Worker manually announce their parent at the end of the `perform`